### PR TITLE
HMAC RequestValidator and GitHub Actions Setup

### PR DESCRIPTION
**Pull request by jpsantosbh**
The RequestValidator class helps to validate Signalwire callback requests.

Added unit tests to confirm both LAML and SWML requests.

Also changes CI from Drone to Github Actions

### CODE REVIEW

File: signalwire/request_validator.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                signature
+            )
+            valid_signature_with_port = compare(
+                self.build_signature(with_port, params),
+                signature
+            )
+
+            return valid_signature_without_port or valid_signature_with_port
+        
+        return self.validate_with_compatibility(uri, params, signature)
```
Comment (iAmmar7) [inline signalwire/request_validator.py:55]: Shouldn't this be a part of `if isinstance(params, str)` condition? Considering the implementation of this feature in [JS SDK](https://github.com/signalwire/signalwire-js/blob/main/packages/web-api/src/validateRequest.ts#L13).

Comment (jpsantosbh) [reply signalwire/request_validator.py:55]: Good catch! fixed

Comment (iAmmar7) [reply signalwire/request_validator.py:55]: Correct me if I am wrong. This implementation is gonna call the `compatibility_validator.validate` even when the `params` is not an instance of a string. But the JS SDK throws an error if it is not a string.

I don't think it's a big deal but I am just highlighting it in terms of consistency across our SDKs.

Comment (jpsantosbh) [reply signalwire/request_validator.py:55]: Yes, when it's a string there is a chance that is a SWML callback that the `compatibility_validator` can not handle. But for the the other cases it's should be able to handle.

**Review by iAmmar7 — APPROVED** [review]

**Pull request merged**